src/imitation/policies/exploration_wrapper.py

```diff
@@ -0,0 +1,64 @@
+"""Wrapper to turn a policy into a more exploratory version."""
+
+from typing import Optional
+
+import numpy as np
+from stable_baselines3.common import vec_env
+
+from imitation.data import rollout
+
+
+class ExplorationWrapper:
+    """Wraps a PolicyCallable to create a partially randomized version.
+
+    This wrapper randomly switches between two policies: the wrapped policy,
+    and a uniformly random one. After each action, the current policy is kept
```

Comment:
Not necessarily uniform, whatever the action spaces `sample()` method is (e.g. for continuous unbounded distributions it's Gaussian).